auth/build.gradle.kts

```diff
@@ -1,3 +1,4 @@
+
 import Config.Libs.Lint.api
 import Config.Plugins.android
```

Comment:
Mmmm, I'd just get rid of these from GitHub. Looks like AS keeps adding them back for some reason.